terraform/s1-agent.tf

```diff
@@ -0,0 +1,118 @@
+provider "aws" {
+  region = var.cluster_region
+}
+
+# ECS Task Definition
+resource "aws_ecs_task_definition" "s1_agent_task_definition" {
+  family                   = "s1-agent-task-definition"
+  requires_compatibilities = ["EC2"]
+  container_definitions = jsonencode([{
+    name = "s1-agent"
```

Comment:
should we allow customers to set a name? if we should then we shouls add a variable for that (var.s1_agent_name)

Reply:
For now we will not allow changing it using vars